Essentials/src/com/earth2me/essentials/Essentials.java

```diff
@@ -820,6 +820,13 @@ public EssentialsTimer getTimer() {
 
     @Override
     public List<String> getVanishedPlayers() {
+        List<String> list = new ArrayList<>();
+        list.addAll(vanishedPlayers);
```

Comment:
This may be replaced by:

```java
return new ArrayList<>(this.vanishedPlayers);
```

Reply:
I agree with this logic. Please do use `unmodifiableList()`.